homeassistant/components/unifiprotect/services.yaml

```diff
@@ -1,34 +1,137 @@
+take_sample:
+  name: Sample Data
+  description: Generates Sample Data from a given UniFi Protect instance. Useful for debugging or reporting issues. Output data is saved to config/ufp_sample.{time}.zip.
```

Comment:
We are working on a generic service for this.
I think we should not add this here right now.

Reply:
Can you link the details?

Reply:
No, I can't; as there is nothing yet.
We concluded this with the Nabu team last week and aiming for adding it soon.
I personally need to for some other things as well.

Reply:
Is the idea just for a new platform or something to let integrations "debug" stuff?

It is probably fine for us to leave these two out for now then, but long term we would like them added back. This service is good for us to troubleshoot "why is my device not working?" or if there is a new feature added to UniFi Protect and one of us (code owners) do not have the device, we can have users send us the data for it. I have been lucky to grab all of the new devices that have dropped recently (UP Sense restock, AI Bullet, and G4 Doorbell Pro), but sometimes users get devices before we can and adding the service instead of requiring them to install the Python library manually is a huge benefit. 

The other one is really only used for benchmarking since we had some CPU issues when we switched to strongly typed rewrite version of the python library. So, it is not nearly as important. 

(do not mistake my tone, I like the idea of centralizing this stuff, mostly just worried it might take too long to implement, great job as always!)

Reply:
Sweet. Timeline as least is very helpful. Waiting 1-2 releases for this is definitely reasonable. Thanks! I am wrapping up another PR right now (still fighting with Websocket reconnects because Protect likes to crash sometimes...), but I will update this one shortly to remove these two services.